docs/index.md

```diff
@@ -93,7 +93,7 @@ or as a development dependency for your project:
 $ npm install --save-dev mocha
 ```
 
-> Mocha currently requires Node.js v6.x or newer.
+> As of v6.0.0, Mocha requires Node.js v6.0.0 or newer.
```

Comment:
this reminds me; we really need to not publish the site on commits to master.  I have no idea how to do that